tests/cdc-wal2json/dml.sql

```diff
@@ -48,3 +48,18 @@ begin;
 update public.payment set amount = 11.99 where amount = 11.95;
 
 commit;
+
+--
+-- Test "is null" transformation in change data capture.
+--
+begin;
+
+-- Disable triggers to prevent automatic refresh of 'last_updated' attribute
+-- when modifying rows in the address table.
+set session_replication_role = replica;
```

Comment:
I didn't quite understand why we need this.

Reply:
In order to avoid noise in the test result comparing, where the timestamp is always different.